DOL.WHD.Section14c.Web/src/modules/components/dolHeader/dolHeaderDirective.test.js

```diff
@@ -0,0 +1,17 @@
+describe('formFooterControls', function() {
+  beforeEach(module('14c'));
+
+  var element, rootScope;
+  beforeEach(function() {
+    element = angular.element('<form-footer-controls/>');
+    inject(function($rootScope, $compile) {
+      rootScope = $rootScope;
+      //$compile(element)(rootScope);
+    });
+  });
+
+  it('invoke directive', function() {
+    rootScope.$digest();
+    expect(element).toBeDefined();
+  });
+});
```

Comment:
This one also looks like it's a copied file.  Just needs to invoke the right directive and this should be good to go.

Reply:
Ahh sorry, that was a complete oversight on my part :/